tests/frontend/compilers/test_xunitary.py

```diff
@@ -401,7 +401,8 @@ def test_unitaries_do_not_match(self, num_pairs):
     @pytest.mark.parametrize("num_pairs", [4, 5, 6, 7])
     def test_unitary_too_large(self, num_pairs):
         """Test exception raised if the unitary is applied to more
-        than just modes [0, 1, 2, 3, ..., num_pairs-1] and [num_pairs, num_pairs+1, ..., 2*num_pairs-1]."""
+        than just modes [0, 1, 2, 3, ..., num_pairs-1] and [num_pairs, num_pairs+1, ..., 2*num_pairs-1].
```

Comment:
Not touching files like this for cosmetic reasons would make the PR easier to read.